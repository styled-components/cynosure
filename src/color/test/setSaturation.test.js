// @flow
import setSaturation from '../setSaturation'

describe('setSaturation', () => {
  it('should update the saturation and return a hex color', () => {
    expect(setSaturation(0.2, '#CCCD64')).toMatchSnapshot()
  })

  it('should update the saturation and return a color with opacity', () => {
    expect(setSaturation(0.75, 'rgba(204,205,100,0.7)')).toMatchSnapshot()
  })

  it('should update the saturation when passed a string', () => {
    expect(setSaturation('0.75', 'rgba(204,205,100,0.7)')).toMatchSnapshot()
  })
})
