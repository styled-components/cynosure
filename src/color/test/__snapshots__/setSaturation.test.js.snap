// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`setSaturation should update the saturation and return a color with opacity 1`] = `"rgba(228,229,76,0.7)"`;

exports[`setSaturation should update the saturation and return a hex color 1`] = `"#adad84"`;

exports[`setSaturation should update the saturation when passed a string 1`] = `"rgba(228,229,76,0.7)"`;
